### DISCUSSION TITLE
A plugin for checking informal text.

### DISCUSSION THREAD

**Discussion by niallmurphy-ie** [retextjs/retext#73] [Ideas] [answered]
I have a use-case for checking informal text with suggested changes to formal.

Is this something I should create a retext plugin for?

**Comment by Murderlon** [comment, marked as answer]
I'm not sure what you mean with "informal" but if you have specific words in mind you could create a plugin for it, or [see if there already is one](https://github.com/retextjs/retext/blob/main/doc/plugins.md).
